src/UI/Menu/Menu.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import styled from 'styled-components';
+import Settings from './Settings';
+import Sound from './Sound';
+import Main from './Main';
+import {Route} from "react-router-dom";
+
+class Menu extends Component{
+  constructor(props) {
+    super(props);
+    this.buttonHandler = props.buttonHandler;
+    this.state = {
+      menu: "main",
+    }
+    this.menuHandler = this.menuHandler.bind(this);
+  }
+
+  menuHandler(element) {
+    this.setState({menu: element});
+  }
+
+  render() {
+    return (
+      <LeftBlock>
+        <GameTitle>
+          <Title>2048</Title>
+          <Description>Join the numbers and get to the <b>2048 tile</b>!</Description>
+        </GameTitle>
+        <Route path='/settings' render={() => <Settings props={this.props} buttonHandler={this.buttonHandler}/>} />
```

Comment:
The proper way here is to use Routes without render prop functions. For instance:

`<Route path='/whatever'>
  <Settings props={this.props} buttonHandler={this.buttonHandler} />
</Route>`